vespalib/src/vespa/vespalib/net/tls/policy_checking_certificate_verifier.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#include "policy_checking_certificate_verifier.h"
+
+namespace vespalib::net::tls {
+
+namespace {
+
+bool matches_single_san_requirement(const PeerCredentials& peer_creds, const RequiredPeerCredential& requirement) {
+    for (auto& provided_cred : peer_creds.dns_sans) {
```

Comment:
const auto &, or is const also automatic ? (multiple places further down as well)

Reply:
`const`ness is auto-deduced, but explicitly having it `const` is usually better. Will add.